lib/pcc_logger.rb

```diff
@@ -0,0 +1,7 @@
+# A basic logger so we can write log messages in a regular format.
+class PCCLogger
+  def self.log(severity, message, druid = '')
+    log_message = Time.now.iso8601 + " " + druid + " " + message
```

Comment:
uh - the rails log doesn't default to using a timestamp?  Or doesn't have config options for such a thing?